_data/resources.yml

```diff
@@ -331,3 +331,6 @@
     - url: http://tenon.io/
       title: "Accessibility Testing API"
       description: "By Tenon.io"
+    - url: https://packagecontrol.io/packages/WAI-ARIA
```

Comment:
I just used the GH like since it redirects automagically when renamed and I noticed the old url to Package Control 404'd